src/core/a-animation.js

```diff
@@ -356,13 +357,14 @@ function getAnimationValues (el, attribute, dataFrom, dataTo, currentValue) {
   var from = {};
   var partialSetAttribute;
   var to = {};
-
-  if (attributeSplit.length === 2) {
+  if (attributeSplit.length === 2 && attributeSplit.indexOf('color') === -1) {
```

Comment:
Alright took your suggestion and pushed another version up with the fallback to isNaN and a isColor function that checks to make sure schemas are defined so that older tests don't fail with this introduction.